pkgs/development/ocaml-modules/letters/default.nix

```diff
@@ -0,0 +1,47 @@
+{
+  alcotest-lwt,
+  buildDunePackage,
+  ca-certs,
+  colombe,
+  containers,
+  fetchFromGitHub,
+  lib,
+  mrmime,
+  rresult,
+  sendmail,
+  tls-lwt,
+  x509,
+}:
+
+let
+  pname = "letters";
+  version = "0.4.0";
+in
+buildDunePackage {
+  inherit pname version;
+  minimalOCamlVersion = "4.03";
+  src = fetchFromGitHub {
+    owner = "oxidizing";
+    repo = pname;
+    rev = version;
```

Comment:
```suggestion
    repo = "letters";
    tag = version;
```

Reply:
What is the rationale behind duplicating the pname?

Reply:
If you have to override pname you break the derivation

Reply:
Fixed.